CenterForOpenScience/ember-osf-web
# [ENG-4417] Remove BsNav

-   Ticket: [ENG-4417]
-   Feature flag: n/a

## Purpose

Remove all usages of BsNav from the ember-osf-web codebase.

## Summary of Changes

1. Remove bs-nav from settings page
2. Remove bs-nav from registries moderation
3. Remove bs-nav from collections moderation
4. Add some data-test selectors

## Screenshot(s)

<img width="602" alt="Screenshot 2023-04-13 at 4 28 22 PM" src="https://user-images.githubusercontent.com/6599111/231876065-323f0ce0-1324-40af-99cd-6768661f2113.png">
<img width="945" alt="Screenshot 2023-04-13 at 3 40 47 PM" src="https://user-images.githubusercontent.com/6599111/231876069-36bf203d-dd7b-4d5f-89bc-903603bb1af2.png">
<img width="391" alt="Screenshot 2023-04-13 at 2 42 11 PM" src="https://user-images.githubusercontent.com/6599111/231876071-dbeab994-3435-44cc-b38e-52ebd82481d2.png">


## Side Effects

In the previous version of the settings page, the active element was not highlighting the same way as it does on osf.io legacy pages (it wasn't highlighting at all, in fact). This version does highlight like legacy, but not for Firefox. The test selector I'm using works everywhere but there. So it's better and not worse, but it's not perfect. I think, since we went five years with nobody noticing, it's probably fine.

## QA Notes

Just check the navigation mentioned above to make sure everything is cool.


[ENG-4417]: https://openscience.atlassian.net/browse/ENG-4417?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ**
